examples/cluster.js

```diff
@@ -13,7 +13,11 @@ import {Cluster, OSM, Vector as VectorSource} from '../src/ol/source.js';
 import {Tile as TileLayer, Vector as VectorLayer} from '../src/ol/layer.js';
 import {boundingExtent} from '../src/ol/extent.js';
 
-const distance = document.getElementById('distance');
+const distanceInput = document.getElementById('distance');
+const distanceNode = document.getElementById('distance-info');
+const minDistanceInput = document.getElementById('min-distance');
+const minDistanceNode = document.getElementById('min-distance-info');
+const numClustersNode = document.getElementById('num-clusters');
```

Comment:
I am always a bit hesitant to approve pull requests that add a lot of code to examples to make config options configurable through forms, so users see how the options affect the result.

My suggestion: rename this example to something like `cluster-configuration`, and create a new `cluster` example that just shows a cluster source with the defaults.

Reply:
Would you say it is better now? The output nodes aren't really necessary.

Reply:
Definitely better, thanks!